tests/unit/logs/self_monitoring/test_sfm.py

```diff
@@ -50,19 +52,19 @@ def test_self_monitoring_context():
             'Dimensions': [{'Name': 'function_name', 'Value': 'my-lambda-function'}],
             'MetricName': 'Kinesis record age',
             'Unit': 'Seconds',
-            'Values': [5]
+            'StatisticValues': {'Maximum': 10, 'Minimum': 5, 'SampleCount': 2, 'Sum': 15},
```

Comment:
Sum seems worthless:P I assume it's required to report those 4, and Avg is calculated from it. 

Does our dashboard show something reasonable after those changes though? Again, if it's sum there, it's worthless, if avg would be okayish, although I'd still add min+max there now that we have them instead.

Reply:
This is the API of this method. Avg is calculated from it. 
Our dashboard works well with this as is! (I was surprised but multi-value push and statistical-value push seem to resolve to same thing underneath) In the dashboard we have min, max, avg